### ISSUE TITLE
awswrangler.s3.read_parquet_table fails on data which was created by spark.saveAsTable

### ISSUE THREAD

**Issue by hanan-vian** [awslabs/aws-data-wrangler#495]
**Describe the bug**
When I try to load the data via read_parquet_table I get an error that magic byte is missing.
ArrowInvalid: Parquet magic bytes not found in footer. Either the file is corrupted or this is not a parquet file.
The table was created by pyspark write.saveAsTable and can be read by pyspark or athena.
maybe wrangler tries to read non parquet files like SUCCESS etc (see below).
when I read with the same table with read_parquet:
`awswrangler.s3.read_parquet(path='path', path_suffix='.parquet')` it works but when I omit the `path_suffix` I get the same error

![image](https://user-images.githubusercontent.com/63995803/103340728-7a70f780-4a8d-11eb-83c1-537525126f44.png)

**To Reproduce**
private data cannot share.
```
awswrangler.s3.read_parquet_table(table='table', database='database')
```

**Comment by igorborgest** [comment]
@hanan-vian Thanks for reporting that. We will address that in the version `2.3.0`.

**Comment by igorborgest** [comment]
Released on version [2.3.0](https://github.com/awslabs/aws-data-wrangler/releases/tag/2.3.0) 🚀
